src/app/component/MyAccountMyWishlist/MyAccountMyWishlist.component.js

```diff
@@ -11,20 +11,93 @@
 
 import { PureComponent } from 'react';
 import PropTypes from 'prop-types';
+import { ProductType } from 'Type/ProductList';
+import ProductCard from 'Component/ProductCard';
+import WishlistItem from 'Component/WishlistItem';
 import './MyAccountMyWishlist.style';
 
-class MyAccountMyWishlist extends PureComponent {
+export default class MyAccountMyWishlist extends PureComponent {
     static propTypes = {
-        // TODO: implement prop-types
+        isLoading: PropTypes.bool,
+        removeAll: PropTypes.func,
+        addAllToCart: PropTypes.func,
+        isWishlistEmpty: PropTypes.bool,
+        wishlistItems: PropTypes.objectOf(ProductType).isRequired
     };
 
+    static defaultProps = {
+        isLoading: false,
+        removeAll: () => {},
+        addAllToCart: () => {},
```

Comment:
Could all of those properties above really be undefined when coming from props?

Reply:
Was written with reuse-ability in mind, but truth is yours `MyAccountMyWishlist` is unlikely to be reused.